packages/kit/test/utils.js

```diff
@@ -111,6 +111,9 @@ export const test = base.extend({
 	in_view: async ({ page }, use) => {
 		/** @param {string} selector */
 		async function in_view(selector) {
+			const box = await page.locator(selector).boundingBox();
+			if (box !== null) return true;
+
```

Comment:
Cool, didn't know there's the `boundingBox` selector. That should make this function a lot simpler and safer. I still don't understand these lines here though. Why are we returning true when the box exists (anywhere on the page)?

I think we could probably simplify the function altogether as:

```js
const box = await page.locator(selector).boundingBox();
const viewport = await page.viewportSize();
return box && viewport && box.top <= viewport.height && box.bottom >= 0
```

I don't think we need to return an object of `top`/`bottom` like before. I don't see it used anywhere.

Reply:
Yeah, the `top`/`bottom` stuff was useful at one point for testing the tests, but is no longer needed. Nothing depends on `width` or `height` so I think this can be even simpler — just `return !!box`. Will update the PR